lib/devicetrust/native/device_windows.go

```diff
@@ -480,6 +476,48 @@ func solveTPMEnrollChallenge(
 	}, nil
 }
 
+func solveTPMAuthDeviceChallenge(
+	challenge *devicepb.TPMAuthenticateDeviceChallenge,
+) (*devicepb.TPMAuthenticateDeviceChallengeResponse, error) {
+	akPath, err := setupDeviceStateDir(os.UserHomeDir)
```

Comment:
I think `os.UserConfigDir` is more appropriate, see https://pkg.go.dev/os#UserConfigDir.

Reply:
I think this would need to be raised as a challenge in the RFD - I don't particularly have strong thoughts about where this directory is located though.

Reply:
I like Tener's suggestion, seems a bit more appropriate. No objections on my part if you want to take it and retcon the change into the RFD later (I don't think the path change has a meaningful effect in the RFD itself)

Reply:
0472d2cfe3